Modules/FindCUDA.cmake

```diff
@@ -734,15 +767,14 @@ if(NOT CUDA_VERSION VERSION_LESS "5.5")
   cuda_find_library_local_first(CUDA_cudart_static_LIBRARY cudart_static "static CUDA runtime library")
   mark_as_advanced(CUDA_cudart_static_LIBRARY)
 endif()
-if(CUDA_cudart_static_LIBRARY)
-  # Set whether to use the static cuda runtime.
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" ON)
-else()
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" OFF)
-endif()
 
-if(CUDA_USE_STATIC_CUDA_RUNTIME)
-  if(UNIX)
+if(NOT CUDA_USE_STATIC_CUDA_RUNTIME STREQUAL OFF)
```

Comment:
Since CUDA_USE_STATIC_CUDA_RUNTIME is a "BOOL" variable you should do:

if(NOT CUDA_USE_STATIC_CUDA_RUNTIME) or if(CUDA_USE_STATIC_RUNTIME) instead of comparing against a string representation of the value (which could be OFF or 0).